packages/trpc/routers/bookmarks.ts

```diff
@@ -721,7 +732,12 @@ export const bookmarksAppRouter = router({
             ),
           )
           .limit(input.limit + 1)
-          .orderBy(desc(bookmarks.createdAt), desc(bookmarks.id)),
+          .orderBy(
+            input.sortOrder === "asc"
+              ? asc(bookmarks.createdAt)
+              : desc(bookmarks.createdAt),
+            desc(bookmarks.id),
+          ),
```

Comment:
Given that this order button is global, we'll need to modify the `searchBookmarks` endpoint as well, and modify `apps/web/lib/hooks/bookmark-search.ts` to pass the sorting order to it as well.

Reply:
Yes, search will also support sorting now.